react/utils/cookie.ts

```diff
@@ -0,0 +1,16 @@
+import { canUseDOM } from 'exenv'
+
+const ONE_YEAR_S = 60 * 60 * 24 * 365
+
+export const setCookie = (
+  key: string,
+  value: string,
+  path = '/',
+  maxAge: number = ONE_YEAR_S
+) => {
+  if (!canUseDOM) {
+    return
+  }
+
+  window.document.cookie = `${key}=${value};path=${path};max-age=${maxAge}`
```

Comment:
Is this erasing all the cookies and setting it only to be `vtex_binding_address` or does this just change the vale of the cookie you are setting?

Reply:
It just add the new cookie.

Reply:
If the cookie is already there, it will change the value.